src/TuxArchitectStores.js

```diff
@@ -12,48 +12,67 @@ var ArchitectChain = function (storeToArchitect) {
 
 //prototype methods
 //itNeeds FUNCTION: registers the input(s) and/or store(s) that the store will need to waitFor, returns this architectChain instance to allow for method chaining, updates "and" method to allow chaining via conjuction
-//@param inputOrStore STRING: input string to be checked against the architecture object for the corresponding store [ALTERNATE OBJECT: if a store is passed in than lookup __registerId__ on the passed in store] [ALTERNATE ARRAY: can pass in an array of inputs and/or stores to be registered, method chaining will still function properly in this case]
+//@param inputOrStore STRING: input string to be checked against the architecture object for the corresponding store or set of stores [ALTERNATE OBJECT: if a store is passed in than lookup __registerId__ on the passed in store] [ALTERNATE ARRAY: can pass in an array of inputs and/or stores to be registered, method chaining will still function properly in this case]
+//can accept any number of input arguments, strings or objects or arrays of strings and/or objects
```

Comment:
Recommend changing second half of the sentence to be '...: strings, objects, or arrays of strings and/or objects'. If a colon isn't appropriate, maybe change it into two sentences.